playbooks/ansible/roles/sit.gpfs/tasks/common/main.yml

```diff
@@ -1 +1,5 @@
 ---
+- name: Add GPFS tools to path
+  copy:
+    dest: /etc/profile.d/gpfs-path.sh
+    content: "PATH=${PATH}:/usr/lpp/mmfs/bin"
```

Comment:
Where do we require the above export?

Reply:
This is required later when samba users are created using `smbpasswd` tool, which is installer here by Storage Scale instead of the regular path in `/usr/sbin`

Reply:
Ok, got it.